spec/tla/run-iterator-validation.sh

```
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Run all TLA+ iterator model specs and verify expected results.
+# Each spec has: correct config (must PASS), bug config (must FAIL), canary config (must FAIL).
+cd "$(dirname "$0")"
+
+RED='\033[0;31m'; GREEN='\033[0;32m'; YELLOW='\033[0;33m'; NC='\033[0m'
+PASS=0; FAIL=0; ERRORS=0
```

Shellcheck (0.11.0)

[warning] FAIL appears unused. Verify use (or export if used externally).

(SC2034)